applications/asset_tracker_v2/src/ext_sensors/Kconfig

```diff
@@ -9,6 +9,29 @@ menuconfig EXTERNAL_SENSORS
 
 if EXTERNAL_SENSORS
 
+config EXTERNAL_SENSORS_IMPACT_DETECTION
+	bool "Impact detection"
+	depends on ADXL372
```

Comment:
I think you could just select ADX372 here. Then customers only need a single config to turn on impact detection.
If you decide to do so, remember to update the documentation that explains how to enable this feature.